auto_tune_vllm/utils/vllm_cli_parser.py

```
@@ -0,0 +1,515 @@
+        self.arguments: Dict[str, CLIArgument] = {}
+        self.sections: Dict[str, List[str]] = {}
+        self._vllm_version: Optional[str] = None
+    
+    def get_vllm_version(self) -> str:
+        """Get the vLLM version."""
+        if self._vllm_version is not None:
+            return self._vllm_version
+        
+        if self.venv_path:
+            cmd = f"source {self.venv_path}/bin/activate && vllm -v"
+            result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)